frontend/src/components/pages/topics/PublishMessagesModal/PublishMessagesModal.tsx

```diff
@@ -60,59 +60,74 @@ export class PublishMessagesModalContent extends Component<Props> {
 
     render() {
         return (
-            <div className="publishMessagesModal">
-                <div style={{ display: 'flex', gap: '1em', flexWrap: 'wrap' }}>
+            <Flex gap={4} flexDirection="column">
```

Comment:
I think we lost the `flexWrap`? Maybe it's better to be verbose just in case

Reply:
@malinskibeniamin the relevant flexWrap moved to line 91